lib/Extension/LanguageServerRename/Adapter/ReferenceFinder/ClassMover/ClassRenamer.php

```diff
@@ -13,18 +13,23 @@
 use Microsoft\PhpParser\ResolvedName;
 use Phpactor\ClassMover\ClassMover;
 use Phpactor\ClassMover\Domain\Name\QualifiedName;
+use Phpactor\Extension\Core\Application\Helper\ClassFileNormalizer;
```

Comment:
would be better not to couple to this class here (most anything here is legacy). Instead you can use the adapter in this package: ClassToFileUriToNameConverter